### DISCUSSION TITLE
Yet another unblock (Not working)

### DISCUSSION THREAD

**Discussion by rod520** [3kh0/ext-remover#385] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
This is probably even worse than the ext-remover, but will work on all computers (if it works). I havent tested it out because I'm too chicken.
It's the developer mode.
[HackShop.pdf](https://github.com/3kh0/ext-remover/files/10383676/HackShop.pdf)

**Comment by Aka-but-nice** [comment]
This is yet the worst bypass I have ever seen

**Reply by NathanielThaBeast** [reply]
fax

**Reply by NathanielThaBeast** [reply]
but ayy lets not hate on manss like dat

**Reply by Aka-but-nice** [reply]
I guess, but this exploit, is for students who have second grade IT's

**Comment by rwg100** [comment]
wow! this is so useful! if only we could actually enable dev mode!

**Reply by rod520** [reply]
I found something new[https://yoodley.com/how-to-bypass-the-administrator-on-chromebook/](bypass admin)
I'm not trying it out though

**Reply by Aka-but-nice** [reply]
Bro, if a site you can find on google has that title, don't go to it. It will tell you to just turn on dev mode, or turn on linux
